readme.md

```diff
@@ -387,3 +387,21 @@ Note that this plugin requires Git 1.9 or higher (because it uses the `--exit-co
 The `gh-pages` module writes temporary files to a `node_modules/.cache/gh-pages` directory.  The location of this directory can be customized by setting the `CACHE_DIR` environemnt variable.
 
 If `gh-pages` fails, you may find that you need to manually clean up the cache directory.  To remove the cache directory, run `node_modules/gh-pages/bin/gh-pages-clean` or remove `node_modules/.cache/gh-pages`.
+
+### Deploying with Github Actions
+
+In order to deploy with Github Actions, you will need to define a user and set the git repository for the process. See the example step below
+
+```
+    # REPLACE "tschaub/gh-pages" IN THE GIT URL BELOW WITH YOUR REPOSITORY'S INFORMATION
+    - name: Deploy with gh-pages
+      run: |
+        git remote set-url origin https://git:${GITHUB_TOKEN}@github.com/tschaub/gh-pages.git
+        npx gh-pages -d build -u "github-actions-bot <support+actions@github.com>"
```

Comment:
I came across gh-pages via the example in the create-react-app docs: https://create-react-app.dev/docs/deployment/#step-2-install-gh-pages-and-add-deploy-to-scripts-in-packagejson

If gh-pages is executed via a named script, the command needs to be modified slightly (note the `--` to pass the additional argument to the wrapped gh-pages call). Maybe this should be included as another example.

```json
"scripts": {
   "deploy": "gh-pages -d build"
}
```

```
npm run deploy -- -u "github-actions-bot <support+actions@github.com>"
```